antora/author.sh

```diff
@@ -1,3 +1,3 @@
-#!/bin/bash -e
+#!/usr/bin/env bash -e
```

Comment:
With the GitHub action failing, and when looking at it in my IDE (IntelliJ) which Shell Check enabled, it provided the following link that explains the problem and the remedy: 

https://github.com/koalaman/shellcheck/wiki/SC2096

Please update all occurrences like the one below.

```suggestion
#!/usr/bin/env bash 
set -e
```

Reply:
No problem, I'll run through and correct these

Reply:
Sorry about the oversight! Should be resolved now (and I also picked up that shell check plugin 😄)